mist-common/src/main/java/edu/snu/mist/common/operators/ApplyStatefulWindowOperator.java

```diff
@@ -92,4 +93,9 @@ public void processLeftData(final MistDataEvent input) {
   public void processLeftWatermark(final MistWatermarkEvent input) {
     outputEmitter.emitWatermark(input);
   }
+
+  @Override
+  public void processLeftCheckpoint(final MistCheckpointEvent input) {
+    outputEmitter.emitCheckpoint(input);
```

Comment:
Don't we checkpoint the state of `ApplyStatefulOperator`?

Reply:
I think we should. I'll fix this.

Reply:
In the comment for this class, it says
```
It is designed to do separate stateful operation over each window input
with the same form of ApplyStatefulFunction.
```

From what this says, this operator is not meant to keep a state(that is the result of previous windows).